evap/staff/views.py

```diff
@@ -569,6 +570,12 @@ def course_edit(request, semester_id, course_id):
 
 @staff_required
 def helper_course_edit(request, semester, course):
+    @receiver(RewardPointGranting.granted_by_removal)
+    def notify_reward_points(users, **kwargs):
+        messages.info(request, ungettext("The removal of participants has granted %(affected)d user reward points.",
+            "The removal of participants has granted %(affected)d users reward points.",
```

Comment:
can you maybe add a comma separated list of the affected user's usernames at the end?

Reply:
Do you want to cap it at a limit? Like `The removal of participants has granted x users reward points: "alfa", "beta", "charlie", "delta", "echo" and 22 additional users` or display all? Someone has explained the worst-case scenario that nearly all participants are affected (like a late-semester event which has been cancelled after all other lectures are open for evaluation and all students were very busy).

Reply:
that's fine, just show them all. it's not a problem if in very special cases the lists can get very long - usually it will only be a short list.